style.css

```diff
@@ -0,0 +1,100 @@
+#id-main-content {
+  box-shadow: 0 0 5px black;
```

Comment:
- [ ] [OPTIONAL] The original design doesn't have a box shadow, please remove it